test/cmd/newapp.sh

```diff
@@ -62,6 +62,7 @@ os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'MY
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_USERNAME'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -o yaml' 'ADMIN_PASSWORD'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample --param MYSQL_PASSWORD=hello -o yaml' 'hello'
+os::cmd::expect_success 'oc new-app -e FOO=BAR -f examples/jenkins/jenkins-ephemeral-template.json -o json | grep FOO'
```

Comment:
When looking for text in output prefer `os::cmd::expect_success_and_(not_)text` rather than `grep`.

When looking for specific output, prefer to inspect the created object in the server (create it, then `oc get ... -o jsonpath ...` so you can be certain you are not picking up some other unrelated `FOO`.